.github/workflows/gradle-plugin.yml

```diff
@@ -17,8 +17,8 @@ jobs:
       fail-fast: false
       matrix:
         os: ['ubuntu-24.04', 'macos-14', 'windows-2022']
-        gradle: ['8.7', '9.0.0']
-        agp: ['8.6.0', '8.9.0', '9.0.0-alpha01']
+        gradle: ['8.13', '9.2.0']
+        agp: ['8.11.2', '8.12.3', '9.0.0-alpha11']
```

Comment:
This should be done on TeamCity as well. Please
- Make the changes inside `compose-teamcity-config` repo inside the branch with the same name
- Open PR on space to review that part too
- Run `SNAPSHOT` build from your branch to make sure that it's green